sdk/core/core-tracing/src/createSpan.ts

```diff
@@ -15,11 +15,11 @@ export interface CreateSpanFunctionArgs {
   /**
    * Package name prefix
    */
-  packagePrefix: string;
+  packagePrefix: string | undefined;
   /**
    * Service namespace
    */
-  namespace: string;
+  namespace: string | undefined;
```

Comment:
I wonder if it would be easier to use the empty string as the sentinel value instead of undefined for these two? Then we don't have to make the interface more complicated

Reply:
Yeah, I could do that. I wasn't really sure if one of them was more "javascript-y" or not so I just picked "no value" :)

Reply:
Went with empty string instead